### PR TITLE
Add block list event to all protocols

### PR DESCRIPTION
**Pull request by dayeon5470**
Issue: https://github.com/petabi/review-database/issues/136

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.19.0] - 2023-09-19
+
+### Added
+
+- Added a 'blockList' event with `dcerpc`, `dns`, `http`, `kerberos`, `ldap`,
```
Comment (minshao) [inline CHANGELOG.md:12]: Might be better to replace "'blockList'" with "\`BlockList\`" for consistency.

File: src/backends/postgres/de.rs [hunk 2]
```diff
@@ -94,7 +94,7 @@ impl<'a, 'de> de::Deserializer<'de> for &'a mut Deserializer<'de> {
             .or_else(|e| {
                 self.input
                     .try_get::<'_, _, NaiveDateTime>(self.index)
-                    .map(|t| t.timestamp_nanos())
+                    .map(|t| t.timestamp_nanos_opt().unwrap_or_default())
```
Comment (minshao) [inline src/backends/postgres/de.rs:97]: Please also update chrono version to 0.4.31 in Cargo.toml file

**Comment by syncpark** [comment]
For cleaner commit log management, please move the Release 0.19.0 to the next PR.

File: src/event/dcerpc.rs [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    pub named_pipe: String,
+    pub endpoint: String,
+    pub operation: String,
+}
+
+impl fmt::Display for BlockListDceRpcFields {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(
+            f,
+            "{},{},{},{},{},{} Text,3",
```
Comment (syncpark) [inline src/event/dcerpc.rs:26]: Please replace the `Text` to the protocol name `DceRpc` to distinguish events.
And other protocols too.

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,13 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.19.0] - 2023-09-19
```
Comment (syncpark) [inline CHANGELOG.md:8]: Please move this Release to the next PR

**Review by syncpark — CHANGES_REQUESTED** [review]

**Comment by syncpark** [comment]
@dayeon5470 
The `EventKind` in Hog and review-database has different element for `BlockList` events as follows.

You have to synchronize the `EventKind` enum definition.

```rust
// in Hog
pub enum EventKind {
    DnsCovertChannel,
    _HttpThreat,
    RdpBruteForce,
    RepeatedHttpSessions,
    Log,
    TorConnection,
    DomainGenerationAlgorithm,
    FtpBruteForce,
    FtpPlainText,
    PortScan,
    MultiHostPortScan,
    NonBrowser,
    LdapBruteForce,
    LdapPlainText,
    ExternalDdos,
    CryptocurrencyMiningPool,
    BlockList(RecordType),
}

// in review-database
pub enum EventKind {
    DnsCovertChannel,
    HttpThreat,
    RdpBruteForce,
    RepeatedHttpSessions,
    Log,
    TorConnection,
    DomainGenerationAlgorithm,
    FtpBruteForce,
    FtpPlainText,
    PortScan,
    MultiHostPortScan,
    NonBrowser,
    LdapBruteForce,
    LdapPlainText,
    ExternalDdos,
    CryptocurrencyMiningPool,
    BlockListConn,
    BlockListDns,
}
```

**Comment by syncpark** [comment]
@dayeon5470 This PR should be merged with review-web. Please add these events to review-web too.

**Review by syncpark — APPROVED** [review]

**Pull request merged**